backendless.d.ts

```diff
@@ -752,6 +752,29 @@ declare module Backendless {
         toJSON(): DataQueryI;
     }
 
+    /**
+     * @public
+     * @class Backendless.JSONUpdateBuilder
+     * @constructor
+     */
+    class JSONUpdateBuilder {
```

Comment:
please cover all the signatures in the `tsd.ts` file

Reply:
+